.github/workflows/runtests.yml

```diff
@@ -201,8 +201,11 @@ jobs:
       - run: python -m pip install --upgrade pip wheel setuptools
 
       # dependencies to install in all Python versions:
+              aesara wurlitzer autowrap lxml pytest z3                \
+
       - run: pip install numpy numexpr matplotlib ipython cython scipy aesara \
-                         wurlitzer autowrap lxml lark                         \
+                         wurlitzer autowrap lxml lark z3                       \
+
```

Comment:
I think this blank line is breaking the YAML file, which is why the tests aren't running.